packages/perseus-editor/src/widgets/interactive-graph-editor/components/interactive-graph-sr-tree.test.tsx

```diff
@@ -0,0 +1,290 @@
+            {
+                role: "div",
+                attributes: [
+                    {name: "description", value: "description1 content"},
+                ],
+            },
+            {
+                role: "div",
+                attributes: [
+                    {name: "description", value: "description2 content"},
+                ],
+            },
+        ]);
+    });
+
+    test("should return an array for element with multiple descriptions", () => {
+        // Arrange
+        const {container} = render(
+            <div>
+                <div aria-describedby="description1 description2">label1</div>
+                <div id="description1">description1 content</div>
+                <div id="description2">description2 content</div>
+            </div>,
+        );
+
+        // Act
+        const result = getAccessibilityAttributes(container);
+
+        // Assert
+        expect(result).toEqual([
+            {
+                role: "div",
+                attributes: [
+                    {name: "description", value: "description1 content"},
+                    {name: "description", value: "description2 content"},
+                ],
+            },
+        ]);
+    });
+
+    test("should return an array for element with multiple descriptions with multiple spaces", () => {
+        // Arrange
+        const {container} = render(
+            <div>
+                <div aria-describedby="description1     description2">
+                    label1
+                </div>
+                <div id="description1">description1 content</div>
+                <div id="description2">description2 content</div>
+            </div>,
+        );
+
+        // Act
+        const result = getAccessibilityAttributes(container);
+
+        // Assert
+        expect(result).toEqual([
+            {
+                role: "div",
+                attributes: [
+                    {name: "description", value: "description1 content"},
+                    {name: "description", value: "description2 content"},
+                ],
+            },
+        ]);
+    });
+
+    test("should include hidden descriptions", () => {
+        // Arrange
+        const {container} = render(
+            <div>
+                <div aria-describedby="description1 description2">label1</div>
+                <div id="description1">description1 content</div>
+                <div id="description2" style={{display: "hidden"}}>
+                    description2 content
+                </div>
+            </div>,
+        );
+
+        // Act
+        const result = getAccessibilityAttributes(container);
+
+        // Assert
+        expect(result).toEqual([
+            {
+                role: "div",
+                attributes: [
+                    {name: "description", value: "description1 content"},
+                    {name: "description", value: "description2 content"},
+                ],
+            },
+        ]);
+    });
```

Comment:
This test still passes if I change the code to use `innerText` instead of `textContent`, which is super weird. That's not how it behaves in practice. Not sure what I'm doing wrong. 🤔